scss-c2s/c2s-results.scss

```diff
@@ -0,0 +1,379 @@
+                  margin-top:42px;
+                  font-size:16px;
+                  line-height: 24px;
+                  &:after {
+                    content:"i";
+                    color:$white;
+                    font-family: serif;
+                    font-size: 24px;
+                    line-height: 30px;
+                    //text-indent: -9999px;
+                    background: #0D60B5; //url(images/info_icon.png) center center no-repeat;
+                    height: 30px;
+                    width: 20px;
+                    display: inline-block;
+                    text-align: center;
+                    position: absolute;
+                    top: -30px;
+                    left: 0;
+                  }
+                }
+              }
+            }
+          }
+          > li {
+            &:last-child {
+              border-bottom:2px solid $grey-2;
+              padding-bottom:30px;
+            }
+          }
+          .device {
+            &--tablet {
+              display:none;
+              position: absolute;
+              right:0;
+              top:0;
+              @include media(tablet) {
+                display:block;
+              }
+            }
+            &--mobile {
+              @include media(tablet) {
+                display:none;
+              }
+            }
+          }
+          .openingTimes {
+            margin: 32px 0 0;
+            tr {
+              &:first-child {
+                display:none;
+              }
+              td, th {
+                font-size:20px;
+                line-height: 20px;
+                padding:5px;
+                font-weight: 400;
+                border:none;
+                &:first-child {
+                  padding-left:0;
+                }
+              }
+              td {
+                width:20px;
+                &:nth-child(3) {
+                  &:before {
+                    content: "-";
+                    margin:0px 10px 0 5px;
+                  }
+                }
+                &:last-child {
+                  text-transform: uppercase;
+                }
+              }
+            }
+            + p {
+              font-size: 16px;
+              color:$grey-3;
+            }
+          }
+        }
+        .searchAgain {
+          margin:0 16px 32px;
+          a {
+            margin-left:32px;
+            position: relative;
+            &:before {
+              content: "\2190";
+              position: absolute;
+              left:-32px;
+            }
+          }
+        }
+        .bankHoliday {
+          .results {
+            li {
+              &.results__item{
+                &--nearest{
+                  @include media(tablet) {
+                    min-height: 630px;
+                  }
+                }
+              }
+              .results {
+                &__item {
+                  @include media(tablet) {
+                    min-height: 630px;
```

Comment:
This minimum height seems a bit big, there is quite a large gap between results. Does it need to be so large?

![screen shot 2018-01-16 at 08 12 58](https://user-images.githubusercontent.com/639706/34978127-1b49c37c-fa95-11e7-9da4-742edb335ea3.png)

Reply:
Well spotted. I forgot to add a conditional class after doing the work on the bank holiday message.